### PR TITLE
NEW(ci) adding .travis.yml CI scripts

### PR DESCRIPTION
**Pull request by jirkadanek**

### CODE REVIEW

**Comment by jirkadanek** [comment]
The Travis run takes ~ 1 min without broker and ~ 1.5 min with broker.

Travis has docker0 interface

```
3: docker0: <BROADCAST,MULTICAST,UP> mtu 1500 qdisc noqueue state UP group default 
    link/ether 02:42:69:94:e7:ca brd ff:ff:ff:ff:ff:ff
    inet 172.17.0.1/16 scope global docker0
       valid_lft forever preferred_lft forever
```

**Comment by jirkadanek** [comment]
There is one complicated part remaining. The tests need to learn on what IP is the broker running. Means the broker will have to announce this somehow. In stdout, and get it with some shell? Or Docker Swarm?

**Comment by jirkadanek** [comment]
@rkubis Hi, I think I am done. I choose the swarm mode... Can you (or us together) have a look? @enkeys mentioning just in case you have some idea having done this for python before, with @pematous.

File: scripts/docker-tests.sh [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+# project directory is mounted under /mnt
+PROJECT="/mnt"
+
+yum install -y centos-release-scl
+yum install -y rh-ruby24 rh-ruby24-ruby-devel
+source scl_source enable rh-ruby24
+
+cd /
+gem install minitest
+gem install qpid-proton/proton-c/bindings/ruby/gem/qpid_proton-0.21.0.gem
```
Comment (rkubis) [inline scripts/docker-tests.sh]: Change 21 to *

**Review by rkubis — APPROVED** [review]

File: scripts/docker-tests.sh (hunk 1 above)
Comment (jirkadanek) [reply scripts/docker-tests.sh]: done

**Pull request merged**